packages/nocodb/src/gateways/socket.gateway.ts

```diff
@@ -14,12 +14,19 @@ function getHash(str) {
   return crypto.createHash('md5').update(str).digest('hex');
 }
 
+const url = new URL(process.env.NC_PUBLIC_URL || `http://localhost:${process.env.PORT || '8080'}/`)
+let namespace = url.pathname
+if (!namespace.endsWith('/')) {
+  namespace = namespace + '/';
+}
```

Comment:
```js
let namespace = url.pathname
namespace += namespace.endsWith("/") ? "" : "/"
```

Reply:
Modified it with the code you suggested.